PDKTSwipableCard.xcodeproj/xcuserdata/robinmalhotra.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be versioned. Could you remove the whole `xcuserdata` from the repository and add it to the gitignore? We should have added created a gitignore file in the first place. 

You could copy this one for instance https://github.com/Produkt/Pelican/blob/master/.gitignore

Reply:
Will do!